Makefile

```diff
@@ -162,24 +164,36 @@ endif
 GCP_STORAGE_BUNDLE ?= cloudera-$(BASE_NAME)-images
 GCP_STORAGE_BUNDLE_LOG ?= cloudera-$(BASE_NAME)-images
 
-define GCP_AMI_REGIONS
-asia-east1,asia-east2,asia-south2,asia-southeast1,australia-southeast1,australia-southeast2,asia-northeast3,europe-west1,europe-west2,europe-west3,europe-west4,us-west1,us-west2,us-west3,europe-west6,us-west4,southamerica-east1,us-central1,europe-north1,europe-central2,northamerica-northeast1,northamerica-northeast2,us-east4,asia-south1,us-east1,asia-northeast2,asia-northeast1,asia-southeast2,southamerica-west1
+# AWS AMI region definitions for the different states
+define AWS_AMI_REGIONS_PHASE1
+us-west-1,us-west-2,eu-central-1
 endef
 
-ifndef AWS_AMI_REGIONS
-define AWS_AMI_REGIONS
-ap-northeast-1,ap-northeast-2,ap-south-1,ap-southeast-1,ap-southeast-2,ap-southeast-3,ca-central-1,eu-central-1,eu-west-1,eu-west-2,eu-west-3,sa-east-1,us-east-1,us-east-2,us-west-1,us-west-2,eu-north-1,eu-south-1,af-south-1,me-south-1,ap-east-1,eu-south-2,eu-central-2,me-central-1
+define AWS_AMI_REGIONS_PHASE2
```

Comment:
Copy jobs should handle when an image is already copied to a region, so I would simply use the original `AWS_AMI_REGIONS`, `AWS_GOV_AMI_REGIONS` and `AZURE_STORAGE_ACCOUNTS` lists as phase 2
This has 2 benefits:

1. We have the list of all regions in a single place
2. It would make sure that we have the image in the phase 1 regions